lib/pointers/ptr.go

```diff
@@ -25,6 +25,16 @@ func Deref[T any](v *T, defaultValue T) T {
 	return defaultValue
 }
 
+// Deref safely dereferences a pointer. If pointer is nil, returns zero value,
+// otherwise returns dereferenced value.
+func DerefZero[T any](v *T) T {
```

Comment:
nit: 
```suggestion
func DerefOrZero[T any](v *T) T {
```

The current name sounds like it _is_ always deref the zero values.

Reply:
Turns out I'm not using this at all, just removed it entirely 😁